cfgov/v1/__init__.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import  # Python 2 only
```

Comment:
Looks like this got in there by mistake. But to that point, we are trying to keep things moving towards Python 3 while we still support 2.